lib/Dialect/HW/HWDialect.cpp

```diff
@@ -79,6 +80,10 @@ void HWDialect::initialize() {
 
   // Register interface implementations.
   addInterfaces<HWOpAsmDialectInterface, HWInlinerInterface>();
+
+  // Attach the InnerRefNamespace interface to the default top level container
+  // op so that children can contain InnerRefNamespaces.
```

Comment:
```suggestion
  // op so that children can contain InnerSymbolTables.
```
(sometimes I put apostrophe after specific technical names ("InnerSymbolTable's"), but don't feel strongly about it, but think IST was meant here)